src/tests/specs/ink/Variables.spec.ts

```diff
@@ -208,9 +207,8 @@ describe("Variables", () => {
     // The Original code used 'CompileString', but since the compilation fails,
     // 'compile' can be used instead.
     compileStoryWithoutRuntime("variable_naming_collision_with_arg");
-
     expect(context.errorMessages).toContainStringContaining(
-      "name has already been used for a function"
+      "name has already been used for a argument to knot"
```

Comment:
That's a good catch, upstream simply looks for `"has already been used"`, which I assume is better. I wasn't fully awake when shuffling code around.